### DISCUSSION TITLE
Dark mode support for language card

### DISCUSSION THREAD

**Discussion by theja-vanka** [anuraghazra/github-readme-stats#3441] [Q&A] [answered]
### Describe the bug

The language card doesn't seem to reflect in dark mode, no matter what I do.

### Screenshots / Live demo link

<div align='center'>
    <picture>
        <source media="(prefers-color-scheme: dark)" srcset="https://github-readme-stats.vercel.app/api?username=theja-vanka&show_icons=true&count_private=true&include_all_commits=true&custom_title=My%20Stats&hide_border=true&theme=github_dark">
        <img src='https://github-readme-stats.vercel.app/api?username=theja-vanka&show_icons=true&count_private=true&include_all_commits=true&custom_title=My%20Stats&hide_border=true' alt='My-Github-Stats'>
    </picture>
    <picture>
        <source media="(prefers-color-scheme: dark)" srcset="https://github-readme-stats.vercel.app/api/top-langs/?username=theja-vanka&custom_title=Extensions&langs_count=3&hide_border=true&hide=html,css,MATLAB&theme=github_dark">
        <img src='https://github-readme-stats.vercel.app/api/top-langs/?username=theja-vanka&custom_title=Extensions&langs_count=3&hide_border=true&hide=html,css,MATLAB' alt='Top-Extensions'>
    </picture>
</div>

**Comment by qwerty541** [comment, marked as answer]
Hey, @theja-vanka! Looks like you have made an error inside `hide` querystring parameter value and all parameters listed after it does not affect on the card. I made this to work by replacing `&hide=html,css,MATLAB` with `&hide=html%23css%23MATLAB` inside your querystring.

```html
<div align='center'>
    <picture>
        <source media="(prefers-color-scheme: dark)" srcset="https://github-readme-stats.vercel.app/api?username=theja-vanka&show_icons=true&count_private=true&include_all_commits=true&custom_title=My%20Stats&hide_border=true&theme=github_dark">
        <img src='https://github-readme-stats.vercel.app/api?username=theja-vanka&show_icons=true&count_private=true&include_all_commits=true&custom_title=My%20Stats&hide_border=true' alt='My-Github-Stats'>
    </picture>
    <picture>
        <source media="(prefers-color-scheme: dark)" srcset="https://github-readme-stats.vercel.app/api/top-langs/?username=theja-vanka&custom_title=Extensions&langs_count=3&hide_border=true&hide=html%23css%23MATLAB&theme=github_dark">
        <img src='https://github-readme-stats.vercel.app/api/top-langs/?username=theja-vanka&custom_title=Extensions&langs_count=3&hide_border=true&hide=html%23css%23MATLAB' alt='Top-Extensions'>
    </picture>
</div>
```

<div align='center'>
    <picture>
        <source media="(prefers-color-scheme: dark)" srcset="https://github-readme-stats.vercel.app/api?username=theja-vanka&show_icons=true&count_private=true&include_all_commits=true&custom_title=My%20Stats&hide_border=true&theme=github_dark">
        <img src='https://github-readme-stats.vercel.app/api?username=theja-vanka&show_icons=true&count_private=true&include_all_commits=true&custom_title=My%20Stats&hide_border=true' alt='My-Github-Stats'>
    </picture>
    <picture>
        <source media="(prefers-color-scheme: dark)" srcset="https://github-readme-stats.vercel.app/api/top-langs/?username=theja-vanka&custom_title=Extensions&langs_count=3&hide_border=true&hide=html%23css%23MATLAB&theme=github_dark">
        <img src='https://github-readme-stats.vercel.app/api/top-langs/?username=theja-vanka&custom_title=Extensions&langs_count=3&hide_border=true&hide=html%23css%23MATLAB' alt='Top-Extensions'>
    </picture>
</div>

**Reply by Miqueas** [reply]
Damn this help me a lot. Thanks!

**Comment by theja-vanka** [comment]
Thank you very much. It worked like a charm !! Keep up the good work.

**Comment by theja-vanka** [comment]
Hey, I just noticed that by adding ignore languages with %23 doesn't hide the language in the card. If you see the output above, HTML is clearly shown despite being added to ignore language.

**Reply by qwerty541** [reply]
@theja-vanka  Looks like this URL works correctly when opening it inside separate browser page, but its behavior changes when using `prefers-color-scheme` feature. We already experienced some trobules with this feature, see https://github.com/anuraghazra/github-readme-stats/discussions/3235. I recommend you to use [GitHub theme context tag](https://github.com/anuraghazra/github-readme-stats#use-githubs-theme-context-tag) instead, this one works well.

```html
<a href="https://github.com/theja-vanka#gh-dark-mode-only"><img src="https://github-readme-stats.vercel.app/api/top-langs/?username=theja-vanka&custom_title=Extensions&langs_count=3&hide_border=true&hide=html,css,matlab&theme=github_dark" /></a>
<a href="https://github.com/theja-vanka#gh-light-mode-only"><img src="https://github-readme-stats.vercel.app/api/top-langs/?username=theja-vanka&custom_title=Extensions&langs_count=3&hide_border=true&hide=html,css,matlab" /></a>
```

<a href="https://github.com/theja-vanka#gh-dark-mode-only"><img src="https://github-readme-stats.vercel.app/api/top-langs/?username=theja-vanka&custom_title=Extensions&langs_count=3&hide_border=true&hide=html,css,matlab&theme=github_dark" /></a>
<a href="https://github.com/theja-vanka#gh-light-mode-only"><img src="https://github-readme-stats.vercel.app/api/top-langs/?username=theja-vanka&custom_title=Extensions&langs_count=3&hide_border=true&hide=html,css,matlab" /></a>
